jest.config.js

```diff
@@ -0,0 +1,179 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+  // All imported modules in your tests should be mocked automatically
+  // automock: false,
+
+  // Stop running tests after `n` failures
+  // bail: 0,
+
+  // Respect "browser" field in package.json when resolving modules
+  // browser: false,
+
+  // The directory where Jest should store its cached dependency information
+  // cacheDirectory: "/private/var/folders/48/p8c3p7l136sd_twzwq4k7s0h0000gn/T/jest_dx",
+
+  // Automatically clear mock calls and instances between every test
+  clearMocks: true,
+
+  // Indicates whether the coverage information should be collected while executing the test
+  collectCoverage: true,
+
+  // An array of glob patterns indicating a set of files for which coverage information should be collected
+  // collectCoverageFrom: null,
+
+  // The directory where Jest should output its coverage files
+  coverageDirectory: "test/coverage",
```

Comment:
Do we need any/all of these?

Reply:
I favour collecting the coverage data. It's not a direct measure of quality, but I think keeping an eye it is better than not.

Reply:
Is the CLI output enough?

Reply:
It's a start: it shows the developer the state of the coverage during development. Would like to get this into CI, but see that as a separate task.

Reply:
Haven't got a problem with CLI output, just concerned about generating files that never get looked at.

Reply:
I use `lcov` files to dig into the details locally so we do want to generate those files, but we could reduce the reporters to:

```
  coverageReporters: [
    "text",
    "lcov",
  ],
```
i.e. drop the `json` and `clover` (XML) formats, at least until they're useful.

Reply:
👍